### ISSUE TITLE
question with QM9_NN_CONV.py

### ISSUE THREAD

**Issue by chenhust1995** [pyg-team/pytorch_geometric#1452]
i make my owndataset with changing  QM9_NN_CONV.py,but the test loss is mantaining upper than three,what is the possible reason?it is because of overfiiting?

**Comment by rusty1s** [comment]
This depends, and I can hardly say anything about that since I do not know your dataset. In general, if the training loss goes down while the test loss remains constant, this is a sign of overfitting or that your training data and test data do not any share similarities.

**Comment by chenhust1995** [comment]
https://github.com/rusty1s/pytorch_geometric/blob/73019c061e80b3b7f5f1174528568959caa9adc3/examples/qm9_nn_conv.py#L74

here，dim=64，dim*dim is a very number，and the first num of my first linear layer is larger than 1000，would it may cause overfitting？thanks for your reply

**Comment by rusty1s** [comment]
Sure, you can try reducing the hidden dimensionality or output dimensionality, *e.g.*:
```python
nn = Sequential(Linear(5, 16), ReLU(), Linear(16, dim * dim)) 
```
or
```python
dim=32
```

**Comment by chenhust1995** [comment]
thanks,and in QM9_NN_CONV.PY,what is the affection of https://github.com/rusty1s/pytorch_geometric/blob/73019c061e80b3b7f5f1174528568959caa9adc3/examples/qm9_nn_conv.py#L51
the transform function

**Comment by rusty1s** [comment]
The first transform collects a specific target out of the 12 targets provided by the QM9 dataset. The second transform creates a complete graph, and the third transform computes distances between each edge in your graph and adds them to the edge features.
